packages/src/Dockerfile.build.rpm-systemd

```diff
@@ -8,10 +8,16 @@ ENV PATH $PATH:/usr/local/go/bin
 ENV ROOT_DIR /root
 ENV SOURCE_DIR /source
 
+# Workaround for mirrorlist.centos.org
+# https://serverfault.com/questions/1161816/mirrorlist-centos-org-no-longer-resolve
+RUN sed -i s/mirror.centos.org/vault.centos.org/g /etc/yum.repos.d/*.repo
+RUN sed -i s/^#.*baseurl=http/baseurl=http/g /etc/yum.repos.d/*.repo
+RUN sed -i s/^mirrorlist=http/#mirrorlist=http/g /etc/yum.repos.d/*.repo
+
 # Packages required for both building and packaging
 #
 # This uses Ruby 3.0 since 2.7 (which still used for Debian builds) does not work with fpm: https://github.com/jordansissel/fpm/issues/2048#issuecomment-1972196104
-RUN yum install -y centos-release-scl scl-utils tar make git rpmdevtools gcc && yum install -y rh-ruby30 rh-ruby30-ruby-devel
```

Comment:
While I was testing, I noticed that running `yum install` twice is not working and second time always fail.
I believe that this is because files in `/etc/yum.repos.d/` are modified with the first run of `yum install` and the workaround will no longer be there.
Making this as one command worked.